kata-deploy/scripts/kata-deploy.sh

```diff
@@ -42,6 +42,11 @@ function install_artifacts() {
 	echo "copying kata artifacts onto host"
 	cp -a /opt/kata-artifacts/opt/kata/* /opt/kata/
 	chmod +x /opt/kata/bin/*
+	if [ "$ENABLE_DEBUG" = true ]; then
+		for shim in ${shims[@]}; do
```

Comment:
I think there is also a non-shim named default config file in that path that should also be added to the list - just plain `configuration.yaml`. Which, messes up your for loop a little, so I'll leave that for you to figure out if there is a nice way to cram it into the loop or if we just do the sed for that one on a separate line.

Reply:
That's true, but taking another look at this, it seems that the default isn't used (from looking at `configure_docker()`). In fact, that function should be looping over `shims` as @brtknr's changes are doing fwics rather than hard-coding the hypervisors since the runtime name can be trivially constructed from the "shim" name.

/cc @egernst.

Reply:
Yes, I left the default untouched since as @jodh-intel points out, its not used by any of the runtimes.